### ISSUE TITLE
Support for iOS assets-library:// files

### ISSUE THREAD

**Issue by smartt** [wkh237/react-native-fetch-blob#43]
It's unclear from the docs whether or not this library can be used to upload images from an iOS user's camera roll.  The React Native camera roll API returns file URIs like, `assets-library://asset/asset.JPG?id=1234-5678-ABCDE-3210&ext=JPG`.  Should I be able to use `RNFetchBlob.wrap()` on such a URI, like:

```JavaScript
RNFetchBlob.fetch('POST', 'http://www.example.com/upload-form', {
    Authorization : "Bearer access-token",
    'Content-Type' : 'multipart/form-data',
  }, [
    // append field data from file path
    {
      name : 'avatar',
      filename : 'avatar.png',
      data: RNFetchBlob.wrap("assets-library://asset/asset.JPG?id=1234-5678-ABCDE-3210&ext=JPG")
    }
...
```

**Comment by wkh237** [comment]
@smartt , thank you for giving suggestion and use case. I'm already working on accessing assets, but there's quite a difference between Android and iOS. I will add this requirement to our `0.7.0` milestone.

**Comment by wkh237** [comment]
Hi @smartt , I've released `0.6.2` which implemented this feature, please check if it work as expected, thank you 😏
